p2p/host.go

```diff
@@ -71,24 +59,18 @@ func NewHost(self *Peer, key libp2p_crypto.PrivKey) (Host, error) {
 	p2pHost, err := libp2p.New(ctx,
 		libp2p.ListenAddrs(listenAddr),
 		libp2p.Identity(key),
-		// libp2p.DisableRelay(),
 		libp2p.EnableNATService(),
 		libp2p.ForceReachabilityPublic(),
 	)
 	if err != nil {
 		return nil, errors.Wrapf(err, "cannot initialize libp2p host")
 	}
-	traceFile := os.Getenv("P2P_TRACEFILE")
 
-	const MaxSize = 2_145_728
 	options := []libp2p_pubsub.Option{
-		libp2p_pubsub.WithPeerOutboundQueueSize(64),
-		libp2p_pubsub.WithMaxMessageSize(MaxSize),
-	}
-	if len(traceFile) > 0 {
-		tracer, _ := libp2p_pubsub.NewJSONTracer(traceFile)
-		options = append(options, libp2p_pubsub.WithEventTracer(tracer))
+		libp2p_pubsub.WithValidateQueueSize(64),
```

Comment:
What are these for?

Reply:
Just parameters to control validation in libp2p